.github/workflows/ci.yml

```
@@ -80,6 +80,7 @@ jobs:
           npm run test:ci
 
   automerge:
+    if: github.event_name == 'pull_request' && github.event.pull_request.user.login == 'dependabot[bot]'
```

```suggestion
    if: >
        github.event_name == 'pull_request' &&
        github.event.pull_request.user.login == 'dependabot[bot]'
```

Last suggestion from me and then it's perfect

---

Didnt know that you can do multiline with >